File: Third_Semester/Computer-Science-Workshop-1(CSW1)/Assignment-06-Wrapper Class/Q09.java
// Question 9 : Write a program that reads to convert int, float, double, and boolean as string types and convert them to respective object types using the valueOf method.

import java.util.Scanner;

public class Q09 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter an integer as a string: ");
        String intString = sc.nextLine();
        Integer intObject = Integer.valueOf(intString);
        System.out.println("Integer Object: " + intObject);
        System.out.print("Enter a float as a string: ");
        String floatString = sc.nextLine();
        Float floatObject = Float.valueOf(floatString);
        System.out.println("Float Object: " + floatObject);
        System.out.print("Enter a double as a string: ");
        String doubleString = sc.nextLine();
        Double doubleObject = Double.valueOf(doubleString);
        System.out.println("Double Object: " + doubleObject);
        System.out.print("Enter a boolean as a string (true or false): ");
        String booleanString = sc.nextLine();
        if (booleanString.equalsIgnoreCase("true") || booleanString.equalsIgnoreCase("false")) {
            Boolean booleanObject = Boolean.valueOf(booleanString);
            System.out.println("Boolean Object: " + booleanObject);
        } else {
            System.out.println("Invalid input for boolean.");
        }
    }
}

File: 5. Fifth_Semester/PYTHON/Minor Assignment 2/Q09.py
user = input("Enter an Integer :")
number = int(user)
remainder = number % 5

match remainder:
    case 0: print(f"The remainder when {number} is divided by 5 is 0.")
    case 1: print(f"The remainder when {number} is divided by 5 is 1.")
    case 2: print(f"The remainder when {number} is divided by 5 is 2.")
    case 3: print(f"The remainder when {number} is divided by 5 is 3.")
    case 4: print(f"The remainder when {number} is divided by 5 is 4.")
    case _: print("Invalid Input!!")